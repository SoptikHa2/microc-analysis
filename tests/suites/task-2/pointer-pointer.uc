 f() {
   var a, b;
   a = null;
   b = alloc a;
   return b;
 }
