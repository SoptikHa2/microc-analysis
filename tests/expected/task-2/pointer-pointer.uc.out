---
test: type suites/task-2/pointer-pointer.uc
---
⟦f(){...}:[1:2]⟧ = () -> ↑↑t2
⟦a[2:8]⟧ = ↑t2
⟦b[2:11]⟧ = ↑↑t2

exit: 0
